test/index.ts

```diff
@@ -14,11 +16,27 @@ function getSum(distributions: Reward[]): BigNumber {
 }
 
 describe('Script: rewards distribution ', () => {
-  it('Sum distributed to voters should be equal to initial amount set to distribute', async () => {
-    const TOTAL_REWARDS_AMOUNT = ethers.utils.parseEther('100');
-    const rewardsObject = await calculateRewardsAmounts(TOTAL_REWARDS_AMOUNT);
-    const rewardsSum = getSum(rewardsObject);
+  if (process.env.ETHEREUM_NETWORK === 'mainnet') {
```

Comment:
The tests under this condition are pointless IMO. To my previous point about using mainnet -- we don't want to be running tests with unpredictable, live data, especially not on mainnet. 

Even if it works (I've heard some companies do it lol), it shouldn't be encouraged at all.

Reply:
Ok I will condition the testing if process.env.ETHEREUM_NETWORK != 'mainnet'